pkgs/servers/http/nginx/modules.nix

```diff
@@ -246,6 +267,7 @@ in
     in {
       src = ngx_pagespeed;
       inputs = [ pkgs.zlib pkgs.libuuid ]; # psol deps
+      allowMemoryWriteExecute = true;
```

Comment:
I think this could default to `true` so it is only disabled in the other cases.

Reply:
If add `allowMemoryWriteExecute = true;` only `lua` modules
```
building all machine configurations...
error: attribute 'allowMemoryWriteExecute' missing, at /home/lf/works/src-nix/nixpkgs/nixos/modules/services/web-servers/nginx/default.nix:727:55

```

Reply:
see https://github.com/NixOS/nixpkgs/pull/85567#discussion_r423031088 on how to implement this.

Reply:
Fixed.